internal/handlers/common/aggregations/stages/group.go

```diff
@@ -77,40 +82,16 @@ func newGroup(stage *types.Document) (aggregations.Stage, error) {
 		}
 
 		if field == "_id" {
+			if doc, ok := v.(*types.Document); ok {
+				if err = validateDocumentExpression(doc); err != nil {
```

Comment:
This can be replaced by new operator right? It should do the same thing as validating document expression

Reply:
Let's keep this for now. But this can be replaced when nested operator can be checked.